Exercise-14/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Regex exercise</title>
+
+    <!-- external fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500&display=swap" rel="stylesheet" />
+
+    <!-- external css -->
+    <link rel="stylesheet" href="assets/css/styles.css" />
+  </head>
+  <body>
+    <!-- header section start -->
+    <header class="logo-container">
+      <img src="assets/Sirius_CDW_Cobranded_PMS2747.svg" alt="sirius-logo" />
+    </header>
+    <!-- header section end -->
+
+    <!-- main section start -->
+    <main>
+      <form action="post" id="paymentForm" class="payment-form">
+        <p class="form-title">BILLING DETAILS</p>
+        <div class="person-name double-input">
```

Comment:
could you user proper naming conventions for class names too like, instead of double-input , input-group

Reply:
I have changed it to input-group